db/migrate/20170307114233_add_collection_to_pages.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+class AddCollectionToPages < ActiveRecord::Migration
+  def change
+    add_reference :pages, :collection, index: true
+    add_foreign_key :pages, :collections
+
+    reversible do |change|
+      change.up do
+        Page::Landing.all.each do |landing_page|
+          key = landing_page.slug.split('/').last
+          key = key.blank? ? 'all' : key
+          collection = Collection.find_by_key(key)
```

Comment:
As it is until now possible to create a landing page without a corresponding collection, could we, to be on the safe side, add a check whether the collection is found here, around the next two statements?